touchforms/formplayer/experiments.py

```diff
@@ -0,0 +1,84 @@
+import laboratory
+import json
+
+
+class FormplayerExperiment(laboratory.Experiment):
+
+    session_id_mapping = {}
+
+    def publish(self, result):
+
+        # if we're starting a new form, we need to store the mapping between session_ids so we can use later
+        if (self.name == "new-form"):
+            control_session_id = json.loads(result.control.value)["session_id"]
+            candidate_session_id = json.loads(result.observations[0].value)["session_id"]
+            FormplayerExperiment.session_id_mapping[control_session_id] = candidate_session_id
+
+        control = result.control
+        # We're only ever returning one of these (I think)
+        candidate = result.observations[0]
+
+        # TODO: How to use this data?
```

Comment:
maybe start by logging to a csv? does the library not come with any stats packages or anything?